backend/src/vscode-youi-events.ts

```diff
@@ -141,9 +145,9 @@ export class VSCodeYouiEvents implements YouiEvents {
             } 
 
             return vscode.window.showInformationMessage(`${successInfoMessage}\nWhat would you like to do with it?`, ...items).then(selection => {
-                if (selection === openInNewWorkspace) {
+                if ((selection === openInNewWorkspace) && (!_.isNil(targetFolderUri))) {
```

Comment:
It seems that I can click on openInNewWorkspace button or on addToWorkspace button and it will do nothing.

Reply:
Changed a bit.